frontend/src/components/ContactUs.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+import { useState, useEffect} from 'react'
+import { TextField, Grid, makeStyles, Paper, Button, Typography } from '@material-ui/core'
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles'
+
+
+const useStyles = makeStyles({
+    root:{
+        '& .MuiFormControl-root':{
+            width: '80%',
+            margin: 8,
+        },
+      
+    },
+    pageSideBar:{
+        margin: 8,
+        padding: 50,
+        backgroundColor: "rgb(238,235,228)",
+        
+    },
+   
+    pageContent:{
+        margin: 8,
+        padding: 110,
+        backgroundColor: "rgb(234,232,224)",
+        
+    },
+    submitButton:{
+        margin: 1,
+        
+    },
+    cancelButton:{
+        margin: 1,
+
+    },
+    sideBar:{
+        fontSize: '1rem',
+        fontWeight: 'bold',
+        cursor: 'pointer',
+        '@media (min-width:1100px)':{
+            fontSize: '1.5rem',
+        },
+
+    },
+
+})
+
+const contactValue = {
+    name: '',
+    email: '',
+    message: '',
+
+}
+
+export default function ContactUs() {
+    const [values, setValues] = useState(contactValue);
+    const classes = useStyles();
+
+    const getInput= e =>{
+        const {name, value } = e.target
+        setValues({
+            ... values,
+            [name]: value
+        })
+    }
```

Comment:
This is a clever way of setting the values we need. I didn't think of doing it this way, but it's nice in that it requires less code than what I've done in the [Profile Editing Tool](https://github.com/Conteo-Inc/conteo/pull/53/files). @Deravadom and I decided to create a hook for each value that needs to be set and move them to frontend/src/utils/profile.tsx 

However we decide to set values in a form, it should be consistent throughout the app (unless there's a good reason to make an exception). If we decide to do it this way, then we definitely should pull out the `name` attribute into a const along with `label` and `value` like I mentioned in Help.tsx

Reply:
I think a combination of the two might be beneficial. I like the choice to wrap the values in the `contactValue` object. That's something I'd like to see in the profile editing tool. Also the name here is a bit misleading. `getInput` actually sets the values.

Reply:
Perhaps change getInput to setInput.